.github/workflows/workflow.yaml

```diff
@@ -168,6 +168,13 @@ jobs:
           terraform_version: ${{ inputs.terraform-version }}
           terraform_wrapper: false
 
+      - name: 'Run: terraform version'
+        shell: bash
+        working-directory: ${{ inputs.working-directory }}
+        run: |
+          set +e
```

Comment:
pretty sure this automatically added by github actions on shell step

Reply:
No, that's `set -e`, which stops on errors. Here we want to continue on error so that we can check the exit status `$?`, i.e. we are flipping the switch again :)